library/Businessprocess/Node.php

```diff
@@ -511,7 +511,8 @@ public function toArray(array $parent = null, $flat = false)
         $data = [
             'name'  => $this->getAlias(),
             'state' => $this->getStateName(),
-            'since' => $this->getLastStateChange()
+            'since' => $this->getLastStateChange(),
+            'in_downtime' => $this->isInDowntime() ? 1 : 0
```

Comment:
For JSON, the result is directly encoded. Why not using `true` / `false` here instead? (i.e. the return value of `isInDowntime`)

Reply:
I've found that isInDowntime returns either 'null' of 'true', not false.

Reply:
Okay :/. But anyhow, I'd still suggest to use `true : false` instead of `1 : 0`. These are valid JSON and anyone working with it would certainly expect this to be boolean. (At least I do)

Reply:
agree, updated.